scripts/partition_manager/partition_manager.rst

```diff
@@ -340,15 +342,9 @@ An error is raised if no partition inside the ``one_of`` dict exists.
       some_span:
          span: [something, exists1]
 
-      # Using 'one_of' as a dict value like this ...
-      some_partition:
-         placement:
-            before: {one_of: [does_not_exist_0, does_not_exist_1, exists1, exists2]}
-
-      # ... is equivalent to:
-      some_partition:
-         placement:
-            before: exists1
+   .. note::
+         ``one_of`` is not valid for use with the :ref:`placement property <partition_manager_placement>`, as ``before`` and ``after`` already check for the first existing partition in their list.
+         It is enough to simply pass a list of partitions into these keywords.
```

Comment:
Does the docs have other examples of `one_of` usage, or should we modify what we had to use `span` or `share_size` as examples instead? 

IMO it has value to show the user the semantics of the map here.

Reply:
It looks like this is the only part of the docs that talk about `one_of`. We can add in an example above with `share_size` if we want to show that along with `span`, but it looks a little redundant after the same example with `span`.

Perhaps we should put a list here that shows the properties that support explicit/implicit `one_of` usage within their keywords?

I.e.

```
Properties that must explicitly define the `one_of` keyword to make use of this functionality are:

- span
- share_size

Properties that contain the functionality of `one_of` by default, and therefore should just use a list of partitions, are:

- placement
```